diff --git a/esm/react-deps.js b/esm/react-deps.js
index 75b1cb485b4e7f3a89b6720cc6511604bd4f57eb..23a567a96ef66034936bbfb04e0787d27d3836fd 100644
--- a/esm/react-deps.js
+++ b/esm/react-deps.js
@@ -1,3 +1,3 @@
 import * as React from 'react';
 export { Fragment, cloneElement, createContext, createElement, forwardRef, isValidElement, useContext, useMemo, useRef, useState } from 'react';
-export { useSyncExternalStore } from 'use-sync-external-store/shim/index.js';
+export const useSyncExternalStore = React.useSyncExternalStore;
